java/src/jmri/implementation/DefaultLogix.java

```diff
@@ -35,7 +35,7 @@ public DefaultLogix(String systemName) {
 
     @Override
     public String getBeanType() {
-        return Bundle.getMessage("BeanNameLogix");
+        return Bundle.getMessage("BeanNameLogix");  // NOI18N
```

Comment:
Isn't this in fact a valid application of I18N using the property bundle? To me this ''NOI18N`` annotation seems out of place.
Or do you mean the value (string) should be "Logix" in all Locales?

Reply:
The NOI18N there means that the string on that line ("BeanNameLogix") does not need to be internationalized.  That's correct; it's already the key to a Bundle, so the resulting string is properly internationalized.

To put it another way, the NOI18N indicates that this line doesn't need to be flagged as (perhaps) needing to be converted to use a Bundle. In this case it's because it's already been done; in other cases it's because the string is an internal key, or part of a log message, or whatever.